torchdata/datapipes/iter/load/fsspec.py

```diff
@@ -33,6 +33,7 @@ def _assert_fsspec() -> None:
         )
 
 
+@functional_datapipe("list_file_by_fsspec")
```

Comment:
```suggestion
@functional_datapipe("list_files_by_fsspec")
```
Based on the discussion elsewhere, let's add a 's'. Thank you so much and sorry for the inconvenience.

Reply:
Note that, you will have to replace all occurrence of this usage in the PR